### ISSUE TITLE
🐛 Notifications panel pending spinner loses focus

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#13357]
> [!NOTE]
> This issue has been rewritten from the original based on later discussion.

### 🐛 Bug and Expected Behaviour

Notification panel focus jumps to top of content whenever the pending spinner appears.  It should be unnecessary to cover the notifications list during periodic polling.

### 🕵️ Details

**Operating system**: Mac OS
**Browser**: Safari 
**Assistive Tech**: VoiceOver 

### 📋 Steps to Reproduce

1. Open notifications panel 
2. tab through notifications
3. wait for the focus to change to Top of notification panel (unprompted) 

### 📸 Screenshot

https://github.com/user-attachments/assets/0f9a453e-ea29-4925-bdb0-7d09d74fe859

### ✅ Acceptance Criteria

- [ ] After panel initially loads, no more pending spinners appear
- [ ] (Optional) If you remove the initial pending spinner, you can also close https://github.com/GCTC-NTGC/gc-digital-talent/issues/12295
- [ ] When the notification polling occurs, focus is not lost even if the list is updated

**Comment by petertgiles** [comment]
I think the issue here is that the pending spinner comes up when it polls and loses the focus, right?  In theory, it shouldn't be necessary to put a spinner up for the polling since the user is already looking a data which is unlikely to change very often.  For that matter, there shouldn't need to be a spinner for the initial load, either, [right](https://github.com/GCTC-NTGC/gc-digital-talent/issues/12295)?

**Comment by esizer** [comment]
> I think the issue here is that the pending spinner comes up when it polls and loses the focus, right? In theory, it shouldn't be necessary to put a spinner up for the polling since the user is already looking a data which is unlikely to change very often. For that matter, there shouldn't need to be a spinner for the initial load, either, [right](https://github.com/GCTC-NTGC/gc-digital-talent/issues/12295)?

That seems reasonable to me 👍

**Comment by BrookeCameron** [comment]
@petertgiles works for me!
